src/main/java/DeleteContactPage.java

```diff
@@ -0,0 +1,30 @@
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+public class DeleteContactPage extends BasePage{
+    public DeleteContactPage(WebDriver driver) {
+        super(driver);
+    }
+
+    @FindBy(id = "check-box-remove-contact")
+    private WebElement checkBox;
+
+    @FindBy(id = "submit-remove")
+    private WebElement buttonDeleteElem;
+
+    @FindBy(xpath = "//*[@class='alert text-center alert-warning']")
+    private WebElement alertText;
+
+    public void clickToCheckBox(){
```

Comment:
`clickCheckbox`

Reply:
There are two ways to write clickOnCheckBox, clickCheckBox